Cython/Compiler/ExprNodes.py

```diff
@@ -9107,6 +9120,27 @@ def may_be_none(self):
 
     gil_message = "Constructing Python class"
 
+    def analyse_annotations(self, env):
+        from .AutoDocTransforms import AnnotationWriter
+        position = self.class_def_node.pos
+        dict_items = []
+        for entry in (e for e in env.entries.values() if e.annotation):
+            strval = AnnotationWriter().write(entry.annotation)
```

Comment:
Latest `master` simplifies this to:
```suggestion
        position = self.class_def_node.pos
        dict_items = []
        for entry in (e for e in env.entries.values() if e.annotation):
            strval = entry.annotation.string
```